src/System.Security.Cryptography.Csp/tests/RSACryptoServiceProviderTests.cs

```diff
@@ -270,6 +277,62 @@ public static void NonExportable_Persisted()
             }
         }
 
+        [Fact]
+        public static void ImportParameters_ExponentTooBig_Throws()
+        {
+            using (var rsa = new RSACryptoServiceProvider())
+            {
+                // Verify that Unix shims and Windows Csp both throws the same exception when large Exponent imported
+                Assert.ThrowsAny<CryptographicException>(() => rsa.ImportParameters(TestData.RsaBigExponentParams));
```

Comment:
This is a discussion point. The changes here and elsewhere was related to throwing WindowsCryptographicException in HR cases. However, I didn't think we want to add the (private class) WindowsCryptographicException for Unix, so I changed a few places to throw just CryptographicException (but with the 'fake' hr embedded). I also didn't think it was appropriate to throw OpenSslException.